.github/workflows/code.yml

```diff
@@ -7,7 +7,10 @@ name: Code
 jobs:
   check-native:
     name: Check native
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
```

Comment:
This will run lints and check as well on all platforms? Perhaps enough to do just tests?

Reply:
Lints and checks *should* be the same across all channels, so could exclude if we're confident enough in that

Reply:
To be honest, running `cargo check` doesn't really make sense anyway, as `cargo test` will find anything `cargo check` would

Reply:
Just went to remove check steps and realised we don't test on wasm. Is there a reason for this? Shall I submit a PR using `wasm-bindgen-test` or similar?

Reply:
Had a look at `wasm-bindgen-test` hence the fallback to `cargo check` and the comment that it should be `test` :rofl: